lib/xlsx/xform/book/workbook-xform.js

```diff
@@ -127,7 +127,7 @@ utils.inherits(WorkbookXform, BaseXform, {
       case 'workbook':
         this.model = {
           sheets: this.map.sheets.model,
-          properties: this.map.workbookPr.model,
+          properties: this.map.workbookPr.model || {},
```

Comment:
Sorry. No, actually I don't know how to add the unit test. 
I just fixed an `.xlsx` file parsed with its `this.map.workbookPr.model` being `undefined` causing an null reference error. 
This because the file itself having a missing property and `exceljs` supposing it have.